decoders/connector/volley-boast/vobo-gp-1/v1.0.0/payload.test.ts

```diff
@@ -265,8 +388,22 @@ describe(`fPort: ${testPayloads.data[10].fport} - ${testPayloads.data[10].name}`
 
     const payload = decoderRun(file_path, { payload: raw_payload });
 
+    test("Output Result", () => {
+        expect(Array.isArray(payload)).toBe(true);
+    });
+
+
+    const adcTemperature = payload.find((item) => item.variable === 'ADC Temperature');
+    const fport = payload.find((item) => item.variable === 'fport');
+    const voboType = payload.find((item) => item.variable === 'voboType');
+    const payloadType = payload.find((item) => item.variable === 'payloadType');
+
     test("Check if data is correct", () => {
-        expect(payload).toStrictEqual({ variable: 'ADC Temperature', value: 26.375, units: 'C' });
+        expect(adcTemperature.value).toBe(26.375);
+        expect(adcTemperature.units).toBe("C")
```

Comment:
[TagoIO data format](https://help.tago.io/portal/en/kb/articles/34-sending-data) expects unit instead of unit**s**